src/app/layout/footer/footer.component.html

```diff
@@ -0,0 +1,35 @@
+<footer class="page-footer grey darken-3">
+  <div class="footer-copyright">
+    <div class="container">
+      <div class="row">
+        <div class="col s2">
+          <a class="grey-text text-lighten-4 right" href="#!">{{ 'links.who did this' | translate }}</a>
+        </div>
+        <div class="col s1 center">|</div>
+        <div class="col s2">
+          <a class="grey-text text-lighten-4 left" href="#!">{{ 'links.contact us' | translate }}</a>
+        </div>
+      </div>
+    </div>
+    <div class="container">
+      <div class="row">
+        <div class="col s9"></div>
+          <div class="col s1">
+            <a class="btn-floating right grey darken-2" href="#!">
+              <i class="fa fa-twitter"></i>
+            </a>
+          </div>
+        <div class="col s1">
+          <a class="btn-floating right grey darken-2" href="#!">
+            <i class="fa fa-facebook"></i>
+          </a>
+        </div>
+          <div class="col s1">
+            <a class="btn-floating right grey darken-2" href="#!">
+              <i class="fa fa-instagram"></i>
+            </a> 
+          </div>
```

Comment:
Still there are some places where the indentation is not quite right, can you take a look to those please?

Reply:
Done